DeepLabV3/sky_segmentation_Deeplabv3.ipynb

```diff
@@ -0,0 +1,702 @@
+{
```

Comment:
<p>Hmm the model seems a bit suboptiomal no? For example, the edge of the roof of the building -- it seems it's a little different from the original picture. Is it a result of resizing? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/Adventures-in-TensorFlow-Lite/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>